Documentation/proxy.md

```diff
@@ -6,6 +6,8 @@ etcd currently supports two proxy modes: `readwrite` and `readonly`. The default
 
 The proxy will shuffle the list of cluster members periodically to avoid sending all connections to a single member.
 
+The list of available client URLs used by proxy is retrieved from client URLs advertised in etcd cluster, which are specified through `-advertise-client-urls` flag when bootstrapping etcd members. So it is critical to set accessible address(es) for proxy in `-advertise-client-urls` flag.
```

Comment:
The member list consists of all client URLs advertised within the cluster, as specified in each members `-advertise-client-urls` flag. If this flag is set incorrectly, a subset of requests sent to the proxy will <what happens?>`

Do these requests hang, wait for a timeout, fail immediately...?
How often is this list of client urls recalculated?